source/_integrations/telegram_bot.markdown

```
@@ -611,6 +611,17 @@ actions:
       disable_web_page_preview: true
 ```
 
+## Example: send_message to a topic within a group:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)